apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SpannerClient.Streaming.cs

```diff
@@ -28,7 +28,7 @@ public partial class SpannerClient
         /// <param name="request"></param>
         /// <param name="callSettings"></param>
         /// <returns></returns>
-        internal virtual AsyncServerStreamingCall<PartialResultSet> ExecuteSqlStream(ExecuteSqlRequest request, CallSettings callSettings = null)
+        public virtual AsyncServerStreamingCall<PartialResultSet> ExecuteSqlStream(ExecuteSqlRequest request, CallSettings callSettings = null)
```

Comment:
Hang on - no, *this* source file isn't generated, as far as I can tell. Yes, it's modifying SpannerClient *most* of which is generated, but I'm fine with that. No problems :)